### PR TITLE
Fix transcriptions in evaluation set

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by stweil** [comment]
These are some examples where Tesseract OCR disagrees with the GT data, and Tesseract is right.

File: xml/NZZ_groundtruth/nzz_18590920_0_0_a1_p1_1.xml [hunk 1]
```diff
@@ -10262,12 +10262,12 @@ Anstalt je länger je mehr auf eigene Füße zu stellen.</Unicode>
                 <Word id="w_w1aab1c31b2b3b1ac57" language="German" custom="readingOrder {index:5;}">
                     <Coords points="996,3497 1246,3497 1246,3528 996,3528"/>
                     <TextEquiv>
-                        <Unicode>Pierre-Francois-¬</Unicode>
+                        <Unicode>Pierre-François-¬</Unicode>
```
Comment (stweil) [inline xml/NZZ_groundtruth/nzz_18590920_0_0_a1_p1_1.xml:10265]: The line break used in the ground truth is inconsistent. Ideally it should use `⸗`, but here either `-` (like after Pierre) or `¬` (the usual line break used by ABBYY / Transkribus) should be used. Which variant is preferred?

Comment (simon-clematide) [reply xml/NZZ_groundtruth/nzz_18590920_0_0_a1_p1_1.xml:10265]: I am not entirely sure what would be the correct solution as the hyphen does not actually come from hyphenation. @pstroe I think we need a guideline here and then also check the consistency over the rest of the transcriptions.
image

**Comment by stweil** [comment]
Only slightly related to this issue:

@pstroe, I noticed today that my line extractors extracted images with a wrong vertical offset for some of the Transkribus PAGE XML files (see https://github.com/OCR-D/format-converters/issues/16). This affects evaluation images, and I expect that it affects training images, too. Fixing that should give even better Tesseract OCR results. How did you produce line images for OCR?

**Comment by stweil** [comment]
Meanwhile I have fixed around 150 words in the evaluation set, and I estimate that maybe twice that number is wrong in the initial evaluation set. For the training data set I expect a similar error rate in the GT texts. It would be interesting how that affects the trained models and the accuracy numbers.

**Comment by stweil** [comment]
The PAGE XML which was generated by Transkribus is not only invalid XML but also contains strange word and line boxes. See https://github.com/Transkribus/TranskribusCore/issues/46 for details.

**Review by simon-clematide — APPROVED** [review]
All good.
